src/components/main/presets/PresetsWrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const PresetsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
default flex-direction is row, if you remove this line there's no difference (optional..)